argo_rollouts/enabling-ha-support-for-argo-rollouts.adoc

```diff
@@ -0,0 +1,21 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="enabling-ha-support-for-argo-rollouts"]
+= Enabling High Availability support for Argo Rollouts
+:context: enabling-ha-support-for-argo-rollouts
+
+toc::[]
+
+Argo Rollouts supports enabling High Availability (HA) in the `RolloutManager` Custom Resource (CR). When you configure HA in Argo Rollouts, the {gitops-title} Operator automatically sets the number of pods to 2 for the Argo Rollouts controller using the `.spec.ha` field in the `RolloutManager` CR. It also activates leader election, allowing the pods to run in an active-passive process. A single pod actively manages rollouts, while the other pod remains in a passive state, ensuring the additional replica provides redundancy and availability if a node fails.
```

Comment:
Is HA an approved acronym? I commented about it later in your PR, but if its the spec you're talking about, best to put it in backticks.

Reply:
Although I don't know if this acronym is approved, I would like to tread lightly and ensure that high availability is used where it is required in the content. Fixed at multiple places in the content.